tests/integration/pox/stacking/direct-and-pooled-increase-3.spec.ts

```diff
@@ -0,0 +1,168 @@
+import { broadcastStackSTX } from "../helpers-direct-stacking";
+import {
+  broadcastDelegateSTX,
+  broadcastDelegateStackIncrease,
+  broadcastDelegateStackSTX,
+  broadcastStackAggregationCommitIndexed,
+} from "../helpers-pooled-stacking";
+import { uintCV } from "@stacks/transactions";
+
+describe("testing direct stacker as pool operator with auto-unlock under epoch 2.1", () => {
+  let orchestrator: DevnetNetworkOrchestrator;
+  let timeline = {
+    epoch_2_0: 100,
+    epoch_2_05: 102,
+    epoch_2_1: 106,
+    pox_2_activation: 109,
+  };
+  const fee = 1000;
+
+  beforeAll(() => {
+    orchestrator = buildDevnetNetworkOrchestrator(getNetworkIdFromEnv());
+    orchestrator.start();
+  });
+
+  afterAll(() => {
+    orchestrator.terminate();
+  });
+
+  it("using direct stacking and delegate-stack-increase,", async () => {
+    const network = new StacksTestnet({ url: orchestrator.getStacksNodeUrl() });
+
+    // Wait for Stacks genesis block
+    await orchestrator.waitForNextStacksBlock();
+    // Wait for block N+1 where N is the height of the next reward phase
+    await waitForNextRewardPhase(network, orchestrator, 1);
+
+    const blockHeight = timeline.pox_2_activation + 1;
+    const cycles = 1;
+
+    // Alice stacks 90m STX
+    let response = await broadcastStackSTX(
+      2,
+      network,
+      80_000_000_000_000,
+      Accounts.WALLET_1,
+      blockHeight,
+      cycles,
+      fee,
+      0
+    );
+    expect(response.error).toBeUndefined();
+
+    // Faucet delegates 999m to Alice address
+    response = await broadcastDelegateSTX(
+      2,
+      network,
+      Accounts.FAUCET,
+      fee,
+      0,
+      999_000_000_000_000,
+      Accounts.WALLET_1
+    );
+    expect(response.error).toBeUndefined();
+    let [block, tx] = await waitForStacksTransaction(
+      orchestrator,
+      response.txid
+    );
+    expect(tx.success, tx.result).toBeTruthy();
+
+    // Alice locks 999m as pool operator
+    response = await broadcastDelegateStackSTX(
+      2,
+      network,
+      Accounts.WALLET_1,
+      fee,
+      1,
+      Accounts.FAUCET,
+      999_000_000_000_000,
+      Accounts.WALLET_1,
+      blockHeight,
+      1
+    );
+    expect(response.error).toBeUndefined();
+    [block, tx] = await waitForStacksTransaction(orchestrator, response.txid);
+    expect(tx.success, tx.result).toBeTruthy();
+
+    // Alice commits 999m
+    response = await broadcastStackAggregationCommitIndexed(
+      2,
+      network,
+      Accounts.WALLET_1,
+      fee,
+      2,
+      Accounts.WALLET_1,
+      2
+    );
+    expect(response.error).toBeUndefined();
+
+    [block, tx] = await waitForStacksTransaction(orchestrator, response.txid);
+    expect(tx.success, tx.result).toBeTruthy();
+
+    let poxInfo = await getPoxInfo(network);
+
+    expect(poxInfo.current_cycle.id).toBe(1);
+    // Assert that the next cycle has 990m STX locked
+    expect(poxInfo.next_cycle.stacked_ustx).toBe(1_079_000_000_000_000);
```

Comment:
Is this without the new fix for the SIP?

Reply:
No, this is expected behaviour. The comment is wrong and has been updated.

    // Alice amount was not unlocked because the same 
    // pox address was used as pool address
    // thereby, the total locked of 1079m was above the minimum of 89m.